src/domains/queries.ml

```diff
@@ -127,7 +127,7 @@ struct
     | MustBeProtectedBy _ -> (module MustBool)
     | MustBeAtomic -> (module MustBool)
     | MustBeSingleThreaded -> (module MustBool)
-    | MustBeUniqueThread -> (module MustBool)
+    | IsMultipleThread -> (module MustBool) (* see https://github.com/goblint/analyzer/pull/310#discussion_r700056687 on why this needs to be MustBool *)
```

Comment:
Besides renaming `MustBeUnique` -> `IsMultiple` you also renamed `MustBeUniqueThread` -> `IsMultipleThread`, which I think has nothing to do with this issue or discussion. Was renaming this query accidental?